src/int-chrome/testSetup.ts

```diff
@@ -57,7 +57,7 @@ function formLaunchArgs(
 
   const argsWithDefaults = { ...chromeLaunchConfigDefaults, ...launchArgs };
   for (let k in argsWithDefaults) {
-    launchArgs[k] = argsWithDefaults[k];
+    launchArgs[k] = (argsWithDefaults as any)[k]; // TODO@rob
```

Comment:
TIL [multiline suggestions aren't a thing, somehow](https://github.com/isaacs/github/issues/1452), but

```
Object.assign(launchArgs, argsWithDefaults)
```

should do the trick without casts